main.go

```diff
@@ -66,9 +67,15 @@ func main() {
 		logger.Error("Unable to get stdin", zap.Error(err))
 	}
 
-	// directly assign stdout/err to pass through terminal, if applicable
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
```

Comment:
I just realized this was quite important for server types/mods that detect TTY/console in order to render logs with color codes. Will need to make this behavior conditional on the use of remote console.

Reply:
Adjusted defaults in #57